packages/manager/modules/billing/src/components/split-payment/controller.js

```diff
@@ -0,0 +1,37 @@
+export default class BillingPaymentMethodSplitPaymentCtrl {
+  /* @ngInject */
+  constructor($translate) {
+    this.$translate = $translate;
+  }
+
+  getFormattedStatus(status, tagStatusEnum) {
```

Comment:
`tagStatusEnum` is already available in the component's controller scope (`this.tagStatusEnum`). Do we need to pass it in on each call? :)

Reply:
In fact `tagStatusEnum` is passed to the function as well as `status`. This way I limit objects the method will play with, outside of the method scope. It is more a coding style than a real issue to fix.